source/making_payments/index.html.md.erb

```diff
@@ -0,0 +1,124 @@
+---
+title: Making payments 
+weight: 70
+---
+
+# Making payments
+
+When you make a payment, you will need to supply a ``reference``. This is a
+unique identifier for the payment. If your organisation already has an
+existing identifier for payments, you can use it here.
+
+You will also need to supply a ``return_url``, a URL hosted by your service
+for the user to return to after they have completed payment on GOV.UK Pay. See
+the section below on **Choosing the return URL** for more information.
+
+You will need to store the URL from the Location header/in the ``self``
+section of ``links`` in the JSON body (the same URL is shown in both places).
+This URL contains the GOV.UK Pay ``payment_id`` which uniquely identifies the
+payment. An authenticated GET request to the URL will return information about
+the payment and its status.
+
+>It is important that you do not expose the URL with the ``payment_id``
+publicly, for example as a URL parameter or in an insecure cookie. You should
+store it as a secure cookie or in a database.
+
+You will receive the ``next_url``  to which you should direct the user to
+complete their payment. During the GOV.UK Pay beta, it is only returned in
+response to the initial POST call to create a payment, not on sub. It will
```

Comment:
No idea what “not on sub” is trying to say (“not on subsequent requests”?). In any case, it’s not true: getting a single payment will give you a new one-time `next_url`.

Reply:
No idea, I've just deleted the line.